README.md

```diff
@@ -88,87 +25,69 @@ sierra = true
 
 # Add the following section
 [tool.voyager]
-my_contract = { path = "main.cairo" }
+my_contract = { path = "src/main.cairo" }
```

Comment:
It's important to add here that `my_contract` MUST match the actual name of the contract in the project